GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java

```diff
@@ -882,4 +913,37 @@ public void saveSettings() {
             Log.d(TAG, "Could not save settings to file", e);
         }
     }
+
+    private CursorActivationListener activationListener = new CursorActivationListener() {
```

Comment:
changed it to add the activation listener when the first selectable object is added.